utils/build.mjs

```diff
@@ -36,11 +36,12 @@ export async function esbuild(opts = {}, cwd = process.cwd()) {
     throw new Error(`Failed to load "${configPath}`);
   }
 
-  const entryPoints = opts.bundle
-    ? // When bundling, assume `src/index.ts` is the entrypoint file
-      [path.join(cwd, 'src/index.ts')]
-    : // When not bundling, compile all files referenced by the `tsconfig.json` file
-      tsconfig.fileNames;
+  const entryPoints =
+    (opts.entryPoints ?? opts.bundle)
```

Comment:
The parens here seem incorrect. But also, is this change necessary (assuming the other point about the CLI build script change not being necessary)?

Reply:
Yes I had planned to remove this in another PR as a follow-up. Happy to just do that here.